generated/nidaqmx/constants.py

```diff
@@ -480,30 +452,15 @@ class OverflowBehavior(Enum):
 
 
 class OverwriteMode(Enum):
-    OVERWRITE_UNREAD_SAMPLES = 10252  #: When an acquisition encounters unread data in the buffer, the acquisition continues and overwrites the unread samples with new ones. You can read the new samples by setting **relative_to** to **ReadRelativeTo.MOST_RECENT_SAMPLE** and setting **offset** to the appropriate number of samples.
+    OVERWRITE_UNREAD_SAMPLES = 10252  #: When an acquisition encounters unread data in the buffer, the acquisition continues and overwrites the unread samples with new ones. You can read the new samples by setting **relative_to** to **MOST_RECENT_SAMP** and setting **offset** to the appropriate number of samples.
```

Comment:
`**ReadRelativeTo.MOST_RECENT_SAMPLE**` was correct. `**MOST_RECENT_SAMP**` is incorrect.